apps/ledger-live-desktop/src/renderer/families/cosmos/utils.jsx

```diff
@@ -0,0 +1,41 @@
+import { openModal } from "~/renderer/actions/modals";
+import { LEDGER_VALIDATOR_ADDRESS } from "@ledgerhq/live-common/families/cosmos/utils";
+
+const getAccountBannerProps = (state, account, { t, dispatch }) => {
+  const { display, redelegate, ledgerValidator, validatorSrcAddress } = state;
+
+  const title = redelegate
+    ? t("account.banner.redelegation.title")
+    : t("account.banner.delegation.title");
+  const description = redelegate
+    ? t("account.banner.redelegation.description")
+    : t("account.banner.delegation.description", {
+        asset: "ATOM",
+        commission: ledgerValidator.commission * 100,
```

Comment:
While in theory `ledgerValidator` should never be null or undefined here, I would put a fail-safe since unfortunately we can't rely on TS yet.

Reply:
Done